internal/config/0_common_config.go

```diff
@@ -10,13 +10,15 @@ import (
 func init() {
 	viper.SetDefault("app.name", "provisioning")
 	viper.SetDefault("app.port", 8000)
+	viper.SetDefault("app.version", "v1")
 	viper.SetDefault("logging.level", 1)
 	viper.SetDefault("database.host", "localhost")
 	viper.SetDefault("database.port", 5432)
 	viper.SetDefault("database.loglevel", 1)
 	viper.SetDefault("cloudwatch.enabled", false)
 	viper.SetDefault("prometheus.port", 9000)
 	viper.SetDefault("prometheus.path", "/metrics")
+	viper.SetDefault("openapi.path", "./api/openapi.gen.json")
```

Comment:
We need to add this file to the container (copy to the container in the [Dockerfile](https://github.com/RHEnVision/provisioning-backend/blob/main/build/Dockerfile)), otherwise it's gona be a failure in deployments.